Website/Install/Config/09.04.00.config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+  <nodes configfile="web.config">
+    <node path="/configuration/system.web/compilation" action="update" key="targetFramework" value="4.7.2" collision="ignore" />
+    <node path="/configuration/system.web/httpRuntime" action="update" key="targetFramework" value="4.7.2" collision="ignore" />
```

Comment:
```suggestion
    <node path="/configuration/system.web/httpRuntime" action="updateattribute" name="targetFramework" value="4.7.2" />
```

Reply:
Thanks for the advice, I wasn't really sure and grabbed the wrong example to base my change off of. Update on it's way